frontend/src/utils/Icons/Icons.jsx

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 
+import TheatersIcon from '@material-ui/icons/Theaters';
 import WarningIcon from '@material-ui/icons/Warning';
 import InfoIcon from '@material-ui/icons/Info';
 import ErrorIcon from '@material-ui/icons/Error';
 import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import GradeIcon from '@material-ui/icons/Grade';
+import TurnedInIcon from '@material-ui/icons/TurnedIn';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 
-const getIcon = type =>
-  ({
-    success: <CheckCircleIcon />,
-    error: <ErrorIcon />,
-    info: <InfoIcon />,
-    warning: <WarningIcon />
-  }[type]);
+const ICON_MAP = {
+  logoIcon: <TheatersIcon />,
+  success: <CheckCircleIcon />,
+  error: <ErrorIcon />,
+  info: <InfoIcon />,
+  warning: <WarningIcon />,
+  favorite: <FavoriteIcon id='favorites' />,
+  rated: <GradeIcon id='rated' />,
+  watch: <TurnedInIcon id='watch' />,
+  expandedLess: <ExpandLessIcon />,
+  expandedMore: <ExpandMoreIcon />
+};
+
+const getIcon = type => ICON_MAP[type];
```

Comment:
you can do here 
```suggestion
export default getIcon = type => ICON_MAP[type];
```
and we can remove line 29, also this file should not be Icons.jsx since this is not a component

Reply:
I tried `export default getIcon = type => ICON_MAP[type];` but, it doesn't working, for this reason I did: `export default function getIcon(type) { return ICON_MAP[type];}`